core/flyout.js

```diff
@@ -745,7 +745,8 @@ Blockly.Flyout.prototype.layout_ = function(contents, gaps) {
       if (this.horizontalLayout_) {
         cursorX += tab;
       }
-      block.moveBy((this.horizontalLayout_ && this.RTL) ? -cursorX : cursorX,
+      block.moveBy((this.horizontalLayout_ && this.RTL) ?
+          cursorX + blockHW.width - tab : cursorX,
```

Comment:
I certainly don't disagree with this, but I'm curious if you know how it broke? Was it the variable change? Mostly I want to understand why it is fixed now :)